Formula/l/lapack.rb

```diff
@@ -6,6 +6,13 @@ class Lapack < Formula
   license "BSD-3-Clause"
   head "https://github.com/Reference-LAPACK/lapack.git", branch: "master"
 
+  # we need :github_latest here because otherwise
+  # livecheck picks up spurious non-release tags
+  livecheck do
+    url :stable
+    strategy :github_latest
```

Comment:
```suggestion
  livecheck do
    url :stable
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

The standard regex for Git tags like `1.2.3`/`v1.2.3` excludes the tags like `v3.1-firstgitcommit` and `v3.12-pre`.